server/lib/validation/languages/tok.js

```diff
@@ -0,0 +1,70 @@
+const tokenizeWords = require('talisman/tokenizers/words');
+
+const TRANSLATION_KEY_PREFIX = 'TRANSLATION_KEY:';
+
+// Minimum of words that qualify as a sentence.
+const MIN_WORDS = 1;
+
+// Maximum of words allowed per sentence to keep recordings in a manageable duration.
+const MAX_WORDS = 18;
```

Comment:
How long does it approximately take to read out 18 words out loud?

Reply:
We timed a 18-word sentence with 87 characters (a bit above average in length for 18 words), which took around 7 seconds to record at normal speed. At the fastest and slowest speeds that felt natural to me, my speeds ranged from 5 or 6 seconds on the low end to about 11 or 12 seconds on the high end (including the short silences between reading out the sentence and hitting the play/pause button).

By the way, due its very regular and minimalistic phonology and phonotactics, in Toki Pona counting characters would probably be a more accurate measure of how long a sentence takes to read out. If you want to prevent outliers with many long words and proper nouns, a 90-character limit might be better. But we shouldn't get many of those.

Reply:
I just looked it up, the maximum recording length is 10 seconds. I'll leave it up to you to judge what the best approach is here.

Reply:
I just looked it up, the maximum recording length is 10 seconds. I'll leave it up to you to judge what the best approach is here.

Reply:
After discussing this question with other contributors and gauging opinions, I decided to go with a character limit rather than a word limit. That option clearly has more support. I updated the file accordingly.